### DISCUSSION TITLE
Dualshock 4 issues  with Snes

### DISCUSSION THREAD

**Discussion by markoz75** [darthcloud/BlueRetro#693] [Q&A]
Hi, I can't have my PS4 controller analogue sticks to work with SNES (only dpad moves), I also can't configure a second joypad. It does connect but it does not move. I used the web configurator where the 2 analogue sticks are enabled, but nothing happens.

**Comment by darthcloud** [comment]
Post screenshot of what you did

**Reply by markoz75** [reply]
Hi, I resolved the analogue stick issue, now it works (wrong configuration by me), while I can't make the second pad work, it is connected but doesn't move at all. This is the configuration:

![cofig](https://github.com/darthcloud/BlueRetro/assets/59534882/71441158-634e-4452-97da-09e43d0e95ff)
